src/fight.cpp

```diff
@@ -2895,6 +2895,18 @@ int check_recoil(struct char_data *ch, struct obj_data *gun)
   for (obj = ch->cyberware; obj; obj = obj->next_content)
     if (GET_OBJ_VAL(obj, 0) == CYB_FOOTANCHOR && !GET_OBJ_VAL(obj, 9))
       comp++;
+    }
+      case WEAP_LIGHT_PISTOL:
+      case WEAP_HEAVY_PISTOL:
+      case WEAP_MACHINE_PISTOL:
+      case WEAP_SMG:
+      case WEAP_LMG:
+      case WEAP_SPORT_RIFLE:
+      case WEAP_SNIPER_RIFLE:
+      case WEAP_SHOTGUN:
+      case WEAP_ASSAULT_RIFLE:
+    if (GET_OBJ_VAL(obj, 0) == ARMS_MOD_GYROMOUNT && !GET_OBJ_VAL(obj, 9))
+      comp+= 3;
```

Comment:
So, cyberarm gyromount gives 3 points of recoil comp on every firearm up to LMG, but doesn't give it to MMG, HMG and Assault Cannons. I have a feeling this is a terrible attempt to reflect that fact or even functions remotely as intended.